src/System.Text.Encoding/tests/Fallback/EncoderReplacementFallbackTests.cs

```diff
@@ -109,7 +109,8 @@ public void CreateFallbackBuffer_Fallback_InvalidSurrogateChars_ThrowsArgumentOu
             EncoderFallbackBuffer buffer = new EncoderReplacementFallback().CreateFallbackBuffer();
 
             Assert.Throws<ArgumentOutOfRangeException>("charUnknownHigh", () => buffer.Fallback('a', '\uDC00', 0));
-            Assert.Throws<ArgumentOutOfRangeException>("CharUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
+            Assert.Throws<ArgumentOutOfRangeException>(
+                RuntimeDetection.IsNetFramework ? "CharUnknownLow" : "charUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
```

Comment:
@Clockwork-Muse 

> Well yeah the tests fail. Because the changes aren't in in coreclr yet

what changes you are expecting to see in coreclr? are you expecting we need to change the parameter names while throwing in there? is this expected to get the same behavior as the full framework?  please clarify.